### DISCUSSION TITLE
Org request/invite/join based on phone-no

### DISCUSSION THREAD

**Discussion by zaiby** [clerk/javascript#3010] [Feature Requests]
Hi,

I'm developing a product in which a user in an Org may not have an email and therefore it's important that it's also possible to invite/join based on phone no. I can update the component if approved.

Ways to address this:

1. Email is default and could be hidden and phone input could optionally be turned on, so possible to have both or one of them.
2. Email input/tag is changed to support both email and phone no.
3. Support: email, phone, username

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
